RHEL/7/input/system/network/firewalld.xml

```diff
@@ -103,8 +103,155 @@ separation of runtime and permanent configuration options. It has support for
 both IPv4 and IPv6 firewall settings.
 </rationale>
 <ident cce="RHEL7-CCE-TBD" />
-<!--<oval id="service_firewalld_enabled" />-->
+<oval id="service_firewalld_enabled" />
 </Rule>
+
 </Group><!--<Group id="firewalld_activation">-->
 
+<Group id="ruleset_modifications">
+<title>Strengthen the Default Ruleset</title>
+<description>The default rules can be strengthened. The system
+scripts that activate the firewall rules expect them to be defined
+in configuration files under the <tt>/etc/firewalld/services</tt>
+and <tt>/etc/firewalld/zones</tt> directories.
+<br /><br />
+The following recommendations describe how to strengthen the
+default ruleset configuration file. An alternative to editing this
+configuration file is to create a shell script that makes calls to
+the <tt>firewall-cmd</tt> program to load in rules under the <tt>/etc/firewalld/services</tt>
+and <tt>/etc/firewalld/zones</tt> directories.
+<br /><br />
+Instructions apply to both unless otherwise noted. Language and address
+conventions for regular firewalld rules are used throughout this section.
+</description>
+<warning category="general">The program <tt>firewall-config</tt>
+allows additional services to penetrate the default firewall rules
+and automatically adjusts the <tt>firewalld</tt> ruleset(s).</warning>
+
+<Rule id="set_firewalld_default_zone" severity="medium">
```

Comment:
The particular rule ID here and in the referenced OVAL check below is ```set_firewalld_default_zone```, but in the ```RHT-CCP``` profile add-on there's ```set_firewalld_default_rule``` rule references (leading to ```notchecked``` result).

Change the referenced rule name in ```RHT-CCP``` profile to ```set_firewalld_default_zone```.